### DISCUSSION TITLE
Paddle OCR don't detect some Japanese text on clean pdf scans

### DISCUSSION THREAD

**Discussion by SmallThings-coder** [PaddlePaddle/PaddleOCR#10674] [General]
I'm using Paddle to get text from bunch of pdf files. Most of them in great condition and Google OCR works perfect on them. However in terms of the app working offline, I tried Paddle and sometimes small chunks of text can be missing even on very readable files. It is mostly happens with first words of the line or numbers of the paragraphs. I tried removing noise and increasing contrast but nothing change.

Is there anything can be done or it is just natural shortcoming for the current version?

Example: I need this イ also be recognized.

<img width="153" alt="Screen Shot 2023-08-18 at 12 01 44" src="https://github.com/PaddlePaddle/PaddleOCR/assets/36983484/0d05717d-abe3-4dd6-b2a4-ddafbe9a95e3">

**Comment by tsukytaka** [comment]
You should expand image with height about x2 then call detect text box method.

**Reply by hritikakolkar** [reply]
How to do that please explain in some details I didn't understand

**Reply by HuyNgo229** [reply]
Hi, have u tried to configure the paddleOCR function? If u not yet, u should try to read Parameters Description for more detail
```
ocr = PaddleOCR(use_angle_cls=True, det_db_thresh = 0.4, det_db_box_thresh = 0.5,det_db_unclip_ratio = 1.4, max_batch_size = 32,
                det_limit_side_len = 1000, det_db_score_mode = "slow", dilation = False, lang='japan', ocr_version = "PP-OCRv4")
result = ocr.ocr(img_path, cls=True)
```
I changed some parameters and it worked for me!
